package.json

```diff
@@ -55,6 +55,7 @@
   },
   "dependencies": {
     "@material-ui/core": "^4.5.1",
+    "@material-ui/icons": "^4.5.1",
```

Comment:
Move `@material-ui/icons` to `peerDependencies`

Reply:
added to both `peer-dependencies` and `dependencies`